compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -376,7 +376,7 @@ class OMR_EXTENSIBLE CodeGenerator
    bool buildInterpreterEntryPoint() { return false; }
    void generateCatchBlockBBStartPrologue(TR::Node *node, TR::Instruction *fenceInstruction) { return; }
    bool supportsUnneededLabelRemoval() { return true; }
-   bool allowSplitWarmAndColdBlocks() { return false; }
```

Comment:
Is the goal to remove this query entirely? 

I'm a little surprised to see it flip from disabled to enabled, given most places in this it's been deleted. If this is to influence a downstream project like OpenJ9, shouldn't the code be removed from OpenJ9, then subsequently this PR removes the OMR impl?

Reply:
I believe the goal is to remove it, yes. There are still j9 dependencies on it. We are completing this task in 3 phases. That piece will come later once we can verify that what we've already done is kosher. @mgaudet

Reply:
I guess the thing I don't understand is why this flips sense from false to true though. Your PR message seems to suggest that you had tried to revert it, but it didn't take.

Reply:
So when allowSplitWarmAndColdBlocks returns true, it sets the cold code length to 0. This ensures that the cold code paths will never be taken. I made this simple modification and ran a personal build to ensure that things will pass without using any of the cold code paths. Once that was verified, I went on to remove the cold code paths altogether and reverted allowSplitWarmAndColdBlocks to return false, such that any j9 dependencies can continue to use it for now. @mgaudet